include/storage/io_config.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef IO_CONFIG_HPP
+#define IO_CONFIG_HPP
+
+#include <boost/filesystem/path.hpp>
+
+namespace osrm
+{
+namespace storage
+{
+struct IOConfig
+{
+    IOConfig() = default;
```

Comment:
Why is the default constructor needed? A default constructed `IOConfig` can't do anything, no?